data/assets.toml

```diff
@@ -0,0 +1,122 @@
+# Versioning and Subresource Integrity (SRI) for Academic's JavaScript and CSS dependencies
+
+# When updating the version of an asset below, please also update the corresponding SRI.
+# How to update the SRI for an entry:
+#   echo -n "sha512-" && curl --silent "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_CHTML" | openssl dgst -sha512 -binary | openssl enc -base64 -A
+# Or, the SRI for the appropriate asset can be copied from https://cdnjs.com/ .
+
+# JavaScript
+
```

Comment:
Please add NOTE to mark where is this file from, just in case we may need to update some of the js versions later on.

Reply:
> \#  version = "3.3.1"
> \#  url = "https://cdnjs.cloudflare.com/ajax/libs/jquery/%s/jquery.min.js"

We just need to replace "%s" with version's data.  (don't manage sri)
For example,we'd like to change jquery's version from 3.3.1 to 4.0.1(suppose exist).
Then:
1. Access to: [https://cdnjs.cloudflare.com/ajax/libs/jquery/4.0.1/jquery.min.js"](url)
2. Right click save as. (don't copy all and paste,could causes a newline problem)
3. Replace origin jquery.min.js file in the /website/static/cdnjs.cloudflare.com/jquery.min.js

By the way,too new versions could be unstable.

Reply:
It would be better to provide a tool to update those js files.